tests/transformer/test_try.py

```diff
@@ -0,0 +1,184 @@
+    try:
+        m('try')
+        raise IndentationError('f1')
+    except IndentationError as error:
+        m('except')
+"""
+
+
+@pytest.mark.parametrize(*e_exec)
+def test_RestrictingNodeTransformer__visit_Try__1(
+        e_exec, mocker):
+    """It allows try-except statements."""
+    trace = mocker.stub()
+    e_exec(TRY_EXCEPT)['try_except'](trace)
+
+    trace.assert_has_calls([
+        mocker.call('try'),
+        mocker.call('except')
+    ])
+
+
+TRY_EXCEPT_ELSE = """
+def try_except_else(m):
+    try:
+        m('try')
+    except:
+        m('except')
+    else:
+        m('else')
+"""
+
+
+@pytest.mark.parametrize(*e_exec)
+def test_RestrictingNodeTransformer__visit_Try__2(
+        e_exec, mocker):
+    """It allows try-except-else statements."""
+    trace = mocker.stub()
+    e_exec(TRY_EXCEPT_ELSE)['try_except_else'](trace)
+
+    trace.assert_has_calls([
+        mocker.call('try'),
+        mocker.call('else')
+    ])
+
+
+TRY_FINALLY = """
+def try_finally(m):
+    try:
+        m('try')
+        1 / 0
+    finally:
+        m('finally')
+        return
+"""
+
+
+@pytest.mark.parametrize(*e_exec)
+def test_RestrictingNodeTransformer__visit_TryFinally__1(
+        e_exec, mocker):
+    """It allows try-finally statements."""
+    trace = mocker.stub()
+    e_exec(TRY_FINALLY)['try_finally'](trace)
+
+    trace.assert_has_calls([
+        mocker.call('try'),
+        mocker.call('finally')
+    ])
+
+
+TRY_EXCEPT_FINALLY = """
+def try_except_finally(m):
+    try:
+        m('try')
+        1 / 0
+    except:
+        m('except')
+    finally:
+        m('finally')
+"""
+
+
+@pytest.mark.parametrize(*e_exec)
+def test_RestrictingNodeTransformer__visit_TryFinally__2(
+        e_exec, mocker):
+    """It allows try-except-finally statements."""
+    trace = mocker.stub()
+    e_exec(TRY_EXCEPT_FINALLY)['try_except_finally'](trace)
+
+    trace.assert_has_calls([
+        mocker.call('try'),
+        mocker.call('except'),
+        mocker.call('finally')
+    ])
+
+
+TRY_EXCEPT_ELSE_FINALLY = """
+def try_except_else_finally(m):
+    try:
+        m('try')
+    except:
+        m('except')
+    else:
+        m('else')
+    finally:
+        m('finally')
+"""
+
+
+@pytest.mark.parametrize(*e_exec)
+def test_RestrictingNodeTransformer__visit_TryFinally__3(
+        e_exec, mocker):
+    """It allows try-except-else-finally statements."""
+    trace = mocker.stub()
+    e_exec(TRY_EXCEPT_ELSE_FINALLY)['try_except_else_finally'](trace)
+
+    trace.assert_has_calls([
+        mocker.call('try'),
+        mocker.call('else'),
+        mocker.call('finally')
+    ])
+
+
+EXCEPT_WITH_TUPLE_UNPACK = """
+def tuple_unpack(err):
+    try:
+        raise err
+    except Exception as (a, (b, c)):
+        return a + b + c
+"""
+
+
+@pytest.mark.skipif(
+    IS_PY3,
+    reason="tuple unpacking on exceptions is gone in python3")
+@pytest.mark.parametrize(*e_exec)
+def test_RestrictingNodeTransformer__visit_ExceptHandler(
```

Comment:
As there is another test function in this module for `visit_ExceptHandler` its name should end in `__1`.